4.JQUERY/ncmyongjavascript.js

```diff
@@ -0,0 +1,63 @@
+/*Part 1*/
+function sumOdd() {
+		var m =0;
+		var sumOdd = 0;
+		for (m=0; m<=50; m++) {
+			if (m%2 !=0){
+				sumOdd+=m;
+			}
+		}alert(sumOdd);
+	}
+
+/*Part 2*/
+function gradeAve() {
+		var gPE = document.getElementById('gPE').value;
+		var gMath = document.getElementById('gMath').value;
+		var gSci = document.getElementById('gSci').value;
+		var gEng = document.getElementById('gEng').value;
+		var gFil = document.getElementById('gFil').value;
+
+		var totalGrade = Number(gPE) + Number(gMath) + Number(gSci) + Number(gEng) + Number(gFil);
+		var aveGrade = totalGrade / 5;
+
+		document.getElementById('hilGrade').innerHTML = aveGrade;
+		
+		if (aveGrade<=60) {
```

Comment:
On the part 2 activity it says 

> <60 | F

So grades 0-59 must be F but when we have an average of 60 it will also display F based on the condition. 60 is not less than 60 so it must be D